File: 1-10/2/2.cpp
#include <iostream>
using namespace std;
int main()
{
        long i,t,f=0,s=1,sum=0;

        for(i=0;i<4000000;i++)
        {
             if(s>4000000||f>4000000)
                   break;
             if(s%2==0)
                   sum+=s;             
             t=f;
             f=s;
             s=s+t;
        }
        cout<<"\t"<<sum<<endl;
}

File: 1-10/2/2.py
def func():
	first = 1
	second = 2
	val = 2
	while(1):
	    sum = first + second;
	    first  = second
	    second = sum
	    if(sum % 2 ==0):
	            val+=sum
	    if(sum >= 4000000):
	            break;

	print val
	

if __name__ == '__main__':
	func()
